scripts/PerfHarness/PerfHarness.cs

```diff
@@ -0,0 +1,34 @@
+using System.IO;
+using System.Reflection;
+using Microsoft.Xunit.Performance.Api;
+
+public class PerfHarness
+{
+    public static void Main(string[] args)
+    {
+        using (XunitPerformanceHarness harness = new XunitPerformanceHarness(args))
+        {
+            foreach(var testName in EnumeratePerfTests())
+            {
+                harness.RunBenchmarks(GetTestAssembly(testName));
+            }
+        }
+    }
+
+    private static string GetTestAssembly(string testName)
+    {
+        // Assume test assemblies are colocated/restored next to the PerfHarness.
+        return Path.Combine(
+            Path.GetDirectoryName(Assembly.GetEntryAssembly().Location),
+            $"{testName}.dll"
+        );
+    }
+
+    private static string[] EnumeratePerfTests()
+    {
+        return new [] {
+            "System.Binary.Base64.Tests",
```

Comment:
Longer term this could certainly be updated to do some amount of self-discovery; I just kept that out for now to keep the focus on how the harness API is designed to work. Self-discovery itself can be implemented a number of ways, and that is discussion which I believe warrants its own separate thread. To make sure things are clear, I documented the requirements for adding new tests in the README file that I've also added in this PR.